olive/passes/onnx/transformer_optimization.py

```diff
@@ -21,6 +21,21 @@ class OrtTransformersOptimization(Pass):
     """Optimize transformer based models in scenarios where ONNX Runtime does not apply the optimization at load time.
     It is based on onnxruntime.transformers.optimizer."""
 
+    # To extend following list/map from huggingface config
+    # there is the priority order: NUM_HEADS_NAMES[0] and HIDDEN_SIZE_NAMES[0] are the first choice
+    # which means user can override the value in config file or hf_config.config
+    NUM_HEADS_NAMES = ["num_heads", "num_attention_heads", "n_head", "encoder_attention_heads"]
+    HIDDEN_SIZE_NAMES = ["hidden_size", "d_model", "n_embd"]
+    MODEL_TYPE_MAPPING = {
```

Comment:
>MODEL_TYPE_MAPPING [](http://example.com/codeflow?start=4&length=18)

could we unify these definitions with the hf_mapping?

Reply:
Thanks. I do not realize we have the hf_mappings. Updated now.